src/pages/ProgramPage/ProgramPlanCard/index.tsx

```diff
@@ -56,6 +64,7 @@ const ProgramPlanCard: React.VFC<{
   const { isAuthenticated } = useAuth()
   const { setVisible: setAuthModalVisible } = useContext(AuthModalContext)
   const { program } = useProgram(programId)
+  const { productGiftPlan } = useProductGiftPlan(`ProgramPlan_${programPlan?.id}`)
```

Comment:
NITS: this hook contain quite a few query , you only used productGiftPlan.id, maybe write own hook or something else?
no need to change, just mention..

Reply:
OK 
I think maybe it can be used somewhere, so I retain others field